test/parallel/test-timers-setunreftimeout.js

```diff
@@ -0,0 +1,61 @@
+// Flags: --expose-internals
+
+'use strict';
+
+const common = require('../common');
+
+const { strictEqual } = require('assert');
+const { setUnrefTimeout } = require('internal/timers');
+
+// ERR_INVALID_CALLBACK
+{
+  // It throws ERR_INVALID_CALLBACK when first argument is not a function.
+  common.expectsError(
+    () => setUnrefTimeout(),
+    {
+      code: 'ERR_INVALID_CALLBACK',
+      type: TypeError,
+      message: 'Callback must be a function'
+    }
+  );
+}
+
+// test case for passing the arguments
+{
+  for (let i = 0; i < 4; i++) {
+    const results = [];
+    const inputArgs = [];
+    // set the input argument params
+    for (let j = 0; j <= i; j++) {
+      inputArgs.push(j);
+    }
+
+    // For checking the arguments.length,
+    // callback function should not be arrow fucntion.
+    const timer = setUnrefTimeout(common.mustCall(
+      function(args1, args2, args3, args4) {
+        // check the number of arguments passed to this callback.
+        strictEqual(arguments.length, i + 1,
+                    `arguments.length should be ${i + 1}.` +
+                    `actual ${arguments.length}`
+        );
+        results.push(args1);
+        results.push(args2);
+        results.push(args3);
+        results.push(args4);
+      }
+    ), 1, ...inputArgs);
+
+    const testTimer = setTimeout(common.mustCall(() => {
+      const maxArgsNum = 4;
+      for (let k = 0; k < maxArgsNum; k++) {
+        // Checking the arguments passed to setUnrefTimeout
+        const expect = (k <= i) ? inputArgs[k] : undefined;
+        strictEqual(expect, results[k],
+                    `result ${k} should be ${inputArgs[k]}. actual ${expect}`);
```

Comment:
This should be `` `result ${k} should be ${expect}. actual ${results[k]}` `` right?

Reply:
It's right! Thank you. I'll fix it